x-pack/solutions/security/packages/ecs_data_quality_dashboard/impl/data_quality_panel/data_quality_details/indices_details/pattern/index_check_flyout/historical_results/historical_results_list/historical_result/legacy_historical_check_fields/index.test.tsx

```diff
@@ -113,7 +113,7 @@ describe('LegacyHistoricalCheckFields', () => {
             <TestExternalProviders>
               <EuiMarkdownFormat>{tablesMarkdown}</EuiMarkdownFormat>
             </TestExternalProviders>
-          ).container.innerHTML
+          ).container.children[0].innerHTML
```

Comment:
for some reason the this container started producing an additional `<span>` I didn't spend time debugging why but I checked that it works as expected visually